index.html

```diff
@@ -31,21 +31,45 @@
   </head>
   <body>
     <section id="abstract">
-      <p>This document aims to define WebViews</p>
+      <h2>Abstract</h2>
+      <p>This document aims to define what a <a>WebView</a> is.</p>
     </section>
     <section id="sotd">
+      <p>
+        This document is a work in progress. It is intended to provide clarity on the definition of a WebView, which has been a topic of discussion within the WebView Community Group.
+      </p>
     </section>
-    <section class="introductory">
-      <h2>Web Browser vs. WebView Definition</h2>
+    <section id="introduction">
+      <h2>Introduction</h2>
+      <p>
+        The WebView Community Group has identified a lack of clarity and consensus around the definition of a <a>WebView</a>. This has
+        led to confusion among developers and users alike, as the term is often used inconsistently across different platforms
+        and contexts.
+      </p>
       <p>
-        The WebView CG identified a need to come up with a definition to clearly identify a web browser vs. WebView. This was discussed via email
-        <a href="https://lists.w3.org/Archives/Public/public-webview/2025May/0000.html">here</a>.
+        <strong>Non-goal:</strong> The purpose of this document is to define what a WebView is. It does not attempt to categorize or define different types of WebViews (<a href="https://webview-cg.github.io/usage-and-challenges/#types-of-webviews">see the usages and challenges explainer</a> for that).
+
+        <aside class="note">
+          <p> TODO: Move types from other report here?</p>
+        </aside>
       </p>
+    </section>
+    <section id="informative">
+      <h2>How do WebViews differ from other user agents, such as browsers?</h2>
       <p>
-        Here is my attempt to summarise the discussion. Let me know if I missed anything important from the email discussion in this summary. Note that the goal here is only to define a web browser vs. webview, and not to define different types of webview (<a href="https://github.com/WebView-CG/usage-and-challenges/issues/19">see #19</a> for that).
+        This was discussed via <a href="https://lists.w3.org/Archives/Public/public-webview/2025May/0000.html">e-mail</a>. <a>WebViews</a> are usually locked down and configured by apps embedding them to fit their use case. Sometimes they are used to build fully featured browsers or show general web content.
       </p>
-      <section class="informative">
-        <h3>Use Cases Discussed</h3>
+      <section>
+        <h3>WebView User Experience</h3>
+        <ul>
+          <li>Whether the user can enter their own URL to navigate to</li>
+          <li>"Embedded" (e.g. WKWebView) vs "System" (e.g. Custom Tabs) WebViews, which differ by whether they run in a separate context, or the same context as a web browser on the device</li>
```

Comment:
Could we confirm this line? 

I have a bit of confusion on the wording and think there might be something better.

For example, could it be worded like this:

```suggestion
          <li>"Embedded" Web Views vs "Delegated" Browser Views, which differ by whether they run in a separate context, or the same context as a web browser on the device</li>
```

I might have a misunderstanding but I always thought of System Web View as the native web views provided on the OS, e.g. Android System WebView & iOS WKWebView

I also thought Custom Chrome Tabs and SFSafariViewController were more of a delegation to a browser, even though they may give the appearance of being a part of the app.

I tried rewording it to be agnostic of any specific platform or library.

Reply:
That's a valid point. Calling something like custom tabs "System WebView" is very confusing.